libevmasm/KnownState.h

```diff
@@ -37,8 +37,17 @@
 #pragma clang diagnostic ignored "-Wredeclared-class-member"
 #endif // defined(__clang__)
 
+#if defined(_MSC_VER)
```

Comment:
Perhaps add a comment here?

```
This silences the nodiscard warning in `boost/bimap.hpp` that appeared post automated migration from Circle CI MSVC version 19.28.29337.0 to version 19.28.29913.0.
```

Reply:
In addition to the "nodiscard", we might add "try removing it again at some point".